### PR TITLE
PaperTrail is not compatible with sinatra 2.0 yet

### PR DESCRIPTION
**Pull request by jaredbeck**
In the meantime, this commit fixes our CI.

See https://github.com/sinatra/rack-protection/issues/117

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -23,16 +24,15 @@ end
...
   gem "rspec-rails", "~> 3.5.1"
   gem 'rails-controller-testing'
 
-  # Sinatra stable conflicts with AR5's rack dependency. Sinatra master requires
-  # rack-protection master. Specify exact `ref` so it doesn't break in the future.
-  # Hopefully there'll be a sinatra 2.0 release soon.
-  gem 'sinatra', github: 'sinatra/sinatra', ref: "a7483f48b0a18ba792e642a"
-  gem "rack-protection", github: "sinatra/rack-protection", ref: "7e723a74763bb83989d12"
+  # The AR5 version of PaperTrail is not compatible with sinatra 2 yet.
+  # Contributions welcome.
```
Comment (zzak) [inline Appraisals]: Are the compatibility issues documented somewhere?

I'm wondering if it's something I broke.

Comment (jaredbeck) [inline Appraisals]: Thanks for taking a look. I tried testing against sinatra 2.0.0.beta2 but got something like "url_helpers undefined". If you want to tinker with it, I can make you a branch that reproduces the error.

Comment (zzak) [inline Appraisals]: @jaredbeck sure!

Comment (jaredbeck) [inline Appraisals]: Thanks! See https://github.com/airblade/paper_trail/issues/856

**Pull request merged**